eng/PoliCheckExclusions.xml

```diff
@@ -0,0 +1,3 @@
+<PoliCheckExclusions>
+	<Exclusion Type="FolderPathFull">LINUX_TEST_RESULTS|MACOS_TEST_RESULTS|WINDOWS_TEST_RESULTS</Exclusion>
```

Comment:
If there's a doc describing the syntax of these exclusions, please add a comment linking to it

Reply:
The doc is an internal microsoft sharepoint document, I don't know if I can/should check in a reference to it in the repo

Reply:
Oh well